bundlebuilder/constants.py

```diff
@@ -1,10 +1,10 @@
-SCHEMA_VERSION = '1.0.14'
+SCHEMA_VERSION = '1.0.17'
 
 # Default session configuration.
 
 DEFAULT_SESSION_EXTERNAL_ID_PREFIX = 'ctim-bundle-builder'
 
-DEFAULT_SESSION_SOURCE = 'CTIM Bundle Builder'
+DEFAULT_SESSION_SOURCE = 'Threat Response CTIM Bundle Builder'
```

Comment:
Shouldn't there be 'SecureX' somewhere?

Reply:
I think no since looks like inside our repository titles, library descriptions, and Relay documentation we just use `Threat Response`.